packages/backend/package.json

```diff
@@ -13,12 +13,12 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint '{src,apps,libs,test}/**/*.ts' --fix",
-    "test": "jest --coverage",
+    "test": "jest --coverage --detectOpenHandles",
```

Comment:
let's use detectOpenHandles only while local development, as it slows down the CI